From 8d9bf0b3969b29ce85195140ccb4830af6b4ae68 Mon Sep 17 00:00:00 2001
From: Xinyu Chen <xinyu.chen@nxp.com>
Date: Mon, 20 Feb 2017 11:02:47 +0800
Subject: [PATCH 1/2] Workaround to enable the imxvideoconvert_pxp convertor
 for CSC when doing camera preview. The qgstvideorendersink has bug on the
 QWidget paintEvent sync with the gst main loop event. Which would cause sink
 render failed if there's no paintEvent before that.

Signed-off-by: Xinyu Chen <xinyu.chen@nxp.com>
---
 src/multimediawidgets/qpaintervideosurface.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/multimediawidgets/qpaintervideosurface.cpp b/src/multimediawidgets/qpaintervideosurface.cpp
index affcace..2acd25d 100644
--- a/src/multimediawidgets/qpaintervideosurface.cpp
+++ b/src/multimediawidgets/qpaintervideosurface.cpp
@@ -1467,7 +1467,7 @@ bool QPainterVideoSurface::present(const QVideoFrame &frame)
 
             stop();
         } else {
-            m_ready = false;
+            //m_ready = false;
 
             emit frameChanged();
 
-- 
2.9.3

